docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -17,52 +17,39 @@
     "tags": []
```

Comment:
<p>Here it seems that we're mostly documenting an <code>xarray</code> method. Have we historically been documenting the libraries we use in our dataset/science tutorials ? My understanding was that it's something we would do in a technical tutorial but not really in a science tutorial. I could be wrong !</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>You're correct. My intention was to add more description of what we have been doing in the cell below this description.</p>